covid_graphs/covid_graphs/simulation_report.py

```diff
@@ -0,0 +1,42 @@
+from dataclasses import dataclass
+from pathlib import Path
+from typing import List
+
+from .pb.simulation_results_pb2 import SimulationResults
+
+
+@dataclass
+class SimulationReport():
```

Comment:
I just noticed, I think in python3 you can drop the parentheses (I'm like 88% sure).

Reply:
Done